samples/net/wpan_serial/README.rst

```diff
@@ -0,0 +1,40 @@
+.. _wpanusb_serial-sample:
+
+Application implementing 802.15.4 "serial-radio" protocol
+#########################################################
+
+How to use
```

Comment:
Please take a look at samples/basic/blinky/README.rst for an example of how we document samples.  There should be some introduction about what the sample does and what board requirements there are (and what boards can this sample be built for and run on).  There should also be example build/flash/run instructions (as shown in the blinky sample).